release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1328,6 +1336,28 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
+
+* Previously, the web console was disabled when the authorization type was set to `None` in the cluster authentication configuration. With this update, the web console no longer disables when the authorization type was set to `None`. (link:https://issues.redhat.com/browse/OCPBUGS-46068[*OCPBUGS-46068*])
+
+* Previously, the *MachineConfig Details* tab displayed an error when one or more `spec.config.storage.file` did not include optional data.  With this update, the error no longer occurs and the *Details* tab renders as expected. (link:https://issues.redhat.com/browse/OCPBUGS-44049[*OCPBUGS-44049*])
+
+* Previously, an extra name prop was being passed into resource list page extensions used to list related operands on the *CSV details* page. This caused the operand list to be filtered by the CSV name, which will almost always cause it to be an empty list. With this update, operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-42796[*OCPBUGS-42796*])
+
+* Previously, the *Sample* tab did not show when creating a new ConfigMap with one or more ConfigMap ConsoleYAMLSamples present on the cluster.  After this update, the *Sample* tab shows with one or more ConfigMap ConsoleYAMLSamples present. (link:https://issues.redhat.com/browse/OCPBUGS-41492[*OCPBUGS-41492*])
+
+* Previously, the *Events* page resource type filter incorrectly reported the number of resources when three or more resources were selected.  With this update, the filter always reports the correct number of resources. (link:https://issues.redhat.com/browse/OCPBUGS-38701[*OCPBUGS-38701*])
+
+* Previously, the version number text in the updates graph on the *Cluster Settings* page appeared as black text on a dark background while viewing the page using Firefox in dark mode.  With this update, the text appears as white text. (link:https://issues.redhat.com/browse/OCPBUGS-37988[*OCPBUGS-37988*])
+
+* Previously, *Alerting* pages did not show resource information in their empty state. With this update, resource information is available on the *Alerting* pages. (link:https://issues.redhat.com/browse/OCPBUGS-36921[*OCPBUGS-36921*])
+
+* Previously, the OLM CSV annotation contained unexpected JSON , which was successfully parsed, but then threw a runtime error when attempting to use the resulting value. With this update, JSON values from OLM annotations are validated before use, and errors will be logged, but the console will not fail when unexpected JSON is received in an annotation. (link:https://issues.redhat.com/browse/OCPBUGS-35744[*OCPBUGS-35744*])
+
+* Previously, silenced alerts were visible on the *Overview* page of the {product-title} web console. This occured because the alerts did not include any external labels. With this release, silenced alerts include the external labels so they can be filtered out so they are not viewable. (link:https://issues.redhat.com/browse/OCPBUGS-31367[*OCPBUGS-31367*])
```

Comment:
```suggestion
* Previously, silenced alerts were visible on the *Overview* page of the {product-title} web console. This occurred because the alerts did not include any external labels. With this release, silenced alerts include the external labels so they are filtered out and are not viewable. (link:https://issues.redhat.com/browse/OCPBUGS-31367[*OCPBUGS-31367*])
```